### PR TITLE
Davewillingham updates

### PR DESCRIPTION
**Pull request by dscw**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -2,32 +2,17 @@
...
+tags: bellingham whatcom
 region: North America
-meetup-group:
+meetup-group: OWASP-Bellingham
 
 ---
 
-<div style='color:red;'>
+## OWASP Bellingahm
+Welcome to the Bellingham, Washington, USA, chapter homepage. We're currently setting up for our first meetings and will be meeting virtually for the foreseeable future.  If you're inteerested in speaking please reach out to us and check out the [Speaker Agreement](https://owasp.org/www-policy/legal/speaker-agreement).
```
Comment (joshbuker) [inline index.md]: Minor typo: `inteerested` -> `interested`

File: index.md [hunk 2]
```diff
@@ -2,32 +2,17 @@
 
 layout: col-sidebar
 title: OWASP Bellingham
-tags: example-tag
+tags: bellingham whatcom
 region: North America
-meetup-group:
+meetup-group: OWASP-Bellingham
```
Comment (joshbuker) [inline index.md]: Does this integrate with [Meetup - Bham OWASP](https://www.meetup.com/owasp-bellingham-chapter/) directly? Not sure how the config here works

File: info.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
-* Provide Group Information
+* Currently meeting virtually
+* [Become a Member](https://www.owasp.org/index.php/Membership)
+* [Local Chapter Support](https://owasp.org/donate)
+* [Speaker Agreement](https://owasp.org/www-policy/legal/speaker-agreement)
+* [Code of Conduct](https://owasp.org/www-policy/operational/conferences-events.html)
 
 ### Social Links
-* [Meetup](#)
+* [Meetup - coming soon](#)
```
Comment (joshbuker) [inline info.md]: Looks like this will be: https://www.meetup.com/owasp-bellingham-chapter/ ?

File: info.md [hunk 4]
```diff
@@ -1,7 +1,11 @@
...
```
Comment (joshbuker) [inline info.md]: We should include a link to the TAG Discord, assuming we'll be using that for our initial conversation space.

**Review by joshbuker — CHANGES_REQUESTED** [review]
Minor typo that needs fixed

File: info.md (hunk 3 above)
Comment (joshbuker) [reply info.md]: @latf6711 Hi Lisa, let us know if there's any action items for us when it comes to getting the Meetup group going. I've joined as a normal member atm.

**Comment by joshbuker** [comment]
@dscw Heads up, looks like you have signed Git commits but have yet to add your public key to Github so it can verify them.

File: index.md (hunk 2 above)
Comment (dscw) [reply index.md]: Yeah, it's supposed to link directly if the URL matches

**Review by dscw — COMMENTED** [review]
Spelling and fixing meetup tag

**Pull request merged**